src/js/getLocalStorageMoviesIds.js

```diff
@@ -0,0 +1,16 @@
+class localStorageMovies{
```

Comment:
classes usually are started with a capital letter (LocalStorageService Is better)